app/views/layouts/shared/_navbar_pipe.html.erb

```diff
@@ -1,40 +1,44 @@
-<div class="form-inline mr-auto form-inline-custom">
-  <div class="input-group ml-3 mr-3">
-    <div class="input-group-prepend">
-      <div class="input-group-text icon-color">
+<div class="flex items-center ">
+  <div class="flex flex-wrap items-center mr-4 flex-1">
+    <div class="p-2.5 mr-[-2px] border-solid border-2 border-light-palette-p3 rounded-l-md bg-brand-palette-08">
+      <div class="icon-color color-brand-palette-03">
```

Comment:
Aqui seria bom converter essa class `icon-color` mas precisa ser visto se vai dar certo pois o bootstrap pode sobre escrevendo ou usando algum !important, caso não seja possível mapeamos para fazer a conversão quando for removido o bootstrap